pkgs/development/python-modules/fsspec/default.nix

```diff
@@ -66,9 +67,13 @@ buildPythonPackage rec {
     "fsspec"
   ];
 
+  passthru.updateScript = nix-update-script {
+    attrPath = "python3Packages.${pname}";
+  };
+
```

Comment:
From my point of view it doesn't make sense to add `nix-update-script` to Python modules. `nix-update` worked fine the past for manual updates, as far as I remember, and it also seems to be picked-up for bulk updates.

Reply:
Removed the updateScript passthru.